Documentation/MRTKNuGetPackage.md

```diff
@@ -0,0 +1,66 @@
+
+Follow these instructions to add the Mixed Reality Toolkit as a NuGet package to your project.
+
+1. Download the latest [NuGetForUnity](https://github.com/GlitchEnzo/NuGetForUnity/releases/latest) .unitypackage
+1. Import the package into your Unity project, [instructions](https://docs.unity3d.com/Manual/AssetPackages.html).
+1. Configure NuGet For Unity, click on the menu **Edit > Preferences...**.
+    - On the left, find **NuGet for Unity** tab.
+    - Un-check **Read-Only Package Files**.
+1. In the Unity menu bar, click on **NuGet > Manage NuGet Packages**.
+1. In the Search box, enter `Microsoft.MixedReality.Toolkit`, and choose the package you would like to install.
+    - **Microsoft.MixedReality.Toolkit.Foundation** – The core package for MRTK.
+    - **Microsoft.MixedReality.Toolkit.Examples** – The package that contains all of our examples.
+    - **Microsoft.MixedReality.Toolkit.Extensions** – The package that contains extensions services and/or data providers. (more information).
+    - **Microsoft.MixedReality.Toolkit.Tools** – Contains some of the tooling that comes with MRTK (Build Window, etc).
+
+### Updating MRTK NuGet Packages
+
+Steps 1-3 above will only need to be done once for your project, and the update is a much simpler step. Once newer packages available on NuGet.org (including prerelease), follow these steps:
+
+1. In the Unity menu bar, click on NuGet > Manage NuGet Packages.
+1. Switch to the **Updates** tab.
+    - Check the **Show prerelease** box if you want to get latest prerelease version.
+1. Update the packages desired.
+
+## NuGet Package Considerations
+
+The release of MRTK as NuGet package is a new delivery mechanism being explored and there are a couple of key benefits and considerations one must make when choosing whether to consume the NuGet version of MRTK.
+
+### Upgrading to NuGet from .unitypackage or Source
+
+NuGet package consists of compiled binaries as opposed to loose script files, and as such, the C# script asset identifiers are different. As such, the assets like prefabs in the MRTK package have been updated to reference the appropriate script. A project using the .unitypcakge or Source version of MRTK will have to do the same, and although there is code for it this is not a supported scenario, yet.
+
+### Compiled Binaries vs Source Files
+
+Since the NuGet package contains the compiled binaries instead of scripts, this has two major advantages:
+
+- Reduced compilation time
+- No more 40+ C# Project files in Visual Studio
+
+### Debugging Mixed Reality Toolkit
+
+There are known issues with Unity & Visual Studio Tools for Unity that prevent a PDB from being easily debugged in Visual Studio Debugger. So although the package comes with PDBs and source embedded, debugging the DLLs is possible only if it was locally built (read further). There is a workaround being built as part of [MSBuildForUnity](https://github.com/microsoft/MSBuildForUnity/), more updates on that later.
```

Comment:
"more updates on that later"

Recommend a simple "for more information" reference / link.

Reply:
Also, i am not seeing the workaround in this doc (it is possible i just missed it).